saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/servlet/filter/Saml2WebSsoAuthenticationFilter.java

```diff
@@ -109,6 +110,7 @@ public Authentication attemptAuthentication(HttpServletRequest request, HttpServ
 					"No relying party registration found");
 			throw new Saml2AuthenticationException(saml2Error);
 		}
+		this.setDetails(request, authentication);
```

Comment:
In Spring Security's codebase, there are some places that we still call an instance method using `this`, but we are moving away from it. Would you please remove the `this.` and just use `setDetails(request, authentication)`?